vr-je-palette-picker/palette-picker-front-end
# Testing setup

## What does this PR do?
This PR creates Snapshot tests for all components, changes the environment variable value for the production URL, incorporates some styling and adds a utils directory with a clean data helper function.
### Where should the reviewer start?
Start in Container.js and review the added and removed functionality. Give the new component ProjectPage a gander, and also check out the App component to see the functionality that is no longer there. The snapshot tests are pretty straight-forward.
#### How should this be manually tested?
Run the testing suite, go to the homepage and click on the projects folder icon to see projects being rendered on the /projects route!
#### Any background context you want to provide?
The snapshot test for ColorCard is failing right now due to the random hex code that is generated and passed to the component.
#### What are the relevant tickets?
Closes #21 
Closes #19 
Closes #4 
Closes #10 
Closes #11 
#### Screenshots (if appropriate)
<img width="437" alt="Screen Shot 2019-12-10 at 5 36 19 PM" src="https://user-images.githubusercontent.com/31895658/70581195-9a2c3980-1b73-11ea-8766-18865d350d55.png">

#### Additional notes
